src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -972,7 +972,7 @@ public async Task BreakMode_RudeEdits_DocumentOutOfSync()
             }, _telemetryLog);
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/39271")]
```

Comment:
Is this just because the test was implicitly assuming the old behavior and we need some new way to write the test? (The filed bug doesn't make it clear if we understand the issue here.)

Reply:
The test was introduced recently https://github.com/dotnet/roslyn/pull/38905 with a workaround for the particular issue. The current PR is a step toward a regular solution but it requires more changes in `CommitedSolution.cs` started at https://github.com/dotnet/roslyn/pull/38905.